ui/components/ui/jazzicon/jazzicon.component.js

```diff
@@ -48,7 +48,11 @@ export default class Jazzicon extends PureComponent {
 
   appendJazzicon() {
     const { address, diameter, tokenList } = this.props;
-    const image = iconFactory.iconForAddress(address, diameter, tokenList);
+    const image = iconFactory.iconForAddress(
+      address,
+      diameter,
+      tokenList[address.toLowerCase()],
```

Comment:
I don't think this change is needed anymore right? Now that the static token list has been modified to match the structure of the remote token list, I think the iconFactory can remained unchanged?

Reply:
Previously we were sending in the whole tokenList as part of Token Detection V1;  with this change, we are just passing in the metadata for the specified token.

Reply:
Okay, that makes sense. This does seem like an improvement